packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.tsx

```diff
@@ -28,7 +28,7 @@ type TAuthenticationStatus = Record<
     boolean
 > & { document_status?: DeepRequired<GetAccountStatus>['authentication']['document']['status'] };
 
-const ProofOfAddressContainer = observer(() => {
+const ProofOfAddressContainer = observer(props => {
```

Comment:
Using props here and destructing down is not a good way in my opinion. Since we dont know what props are passed in. By listing the props in the function signature, you make it explicit what props the component expects. TypeScript will give an error if you try to destructure a prop that doesn't exist. Plus we wont know what is used and unused in this manner